examples/bench_mark/README.md

```diff
@@ -0,0 +1,26 @@
+# Sidecar demo  
+
+## Description
+
+This demo creates a dApp table that contains two nodes, a ping and a pong, that exchange messages with each other.
```

Comment:
I think you can better explain what this app does. Make it "send" and "reciever" instaead of ping pong, and explain that one just send message and the other just reads it.